app/controllers/application_controller.rb

```diff
@@ -187,6 +187,7 @@ def error_handler(e)
             e.message
           end
 
+    msg = "#{msg}\n#{e.backtrace.join("\n")}" unless MiqEnvironment::Command.is_production?
```

Comment:
Can you please explain what's the purpose of this piece of code?

Reply:
We want to output the s stack trace of the error to the page, but only in dev mode.

Reply:
I will move it to a new PR

Reply:
moved to a new PR https://github.com/ManageIQ/manageiq-ui-classic/pull/7296